examples/axis-annotation/index.js

```diff
@@ -0,0 +1,61 @@
+class Annotation {
+    constructor(value, velocity) {
+        this.value = value;
+        this.velocity = velocity;
+    }
+
+    animate() {
+        this.velocity += (Math.random() - 0.5) * 0.01;
+        this.value += this.velocity;
+        if (this.value < scale.domain()[0]) {
+            this.value = scale.domain()[0] + (scale.domain()[0] - this.value);
+            this.velocity *= -1;
+        }
+        if (this.value > scale.domain()[1]) {
+            this.value = scale.domain()[1] + (scale.domain()[1] - this.value);
+            this.velocity *= -1;
+        }
+    }
+
+    valueOf() {
+        return this.value;
+    }
+}
+
+// Using a value wrapper prevents the identity-based data
+// join within axis churning DOM nodes. If your annotation
+// does not change value, there is no need for the wrapper.
+const annotation = new Annotation(45, 0.01);
+
+const scale = d3.scaleLinear().domain([0, 100]);
+
+const axis = fc
+    .axisBottom(scale)
+    .tickValues([...scale.ticks(), annotation])
+    .decorate(g => {
+        g.filter(d => d === annotation)
+            .select('path')
```

Comment:
should this use `enter()` to ensure that these style / attributes are only set when a new element is created?

Reply:
I could for the styles but the axis code sets the `d` attribute on each update so I figured I'd just lump them together for simplicity